documentation/style.css

```diff
@@ -0,0 +1,50 @@
+/* Main section */
+main{
+    font-size: 1.2rem;
+    text-align: center;
+}
+
+/* Class Names Style*/
+.class-names{
+    font-style:italic;
+    color:var(--class-color);
+}
+
+/*Paragraph */
+main p{
+    margin:0.8rem auto;
+    text-align:left;
+    width: 60%;
+}
+
+
+/* Examples container */
+
+.example-container{
+    align-items: center; 
+    border:2px;
+    border-radius: 12px;
+    border-style: solid;
+    display:flex;
```

Comment:
Suggestion: Flex and positioning using flex properties like centering etc can be moved into utility classes as they will be used repeatedly in the upcoming components as well. So by that way, you can avoid rewriting the same bunch of properties

Reply:
Thankyou for your suggestion. I have moved the common classes to the utils file. I have not added them to my components since then the number of classnames added to those components would create confusion but I have used them in my documentation site part. Please let me know if you have any other solution or if it is ok to merge now.

Reply:
Looks good to merge @mnath30 👍